mock-recipe-server/api_index.html

```diff
@@ -148,11 +148,9 @@
         let url = new URL(`/${testcase.dataset.name}/api/v1`, window.location);
         testcase.querySelector('.url').textContent = url.href;
         testcase.querySelector('.browser-code').value = (
-          `Cu.import('resource://shield-recipe-client/lib/Storage.jsm');\n` +
-          `Storage.clearAllStorage();\n` +
-          `Services.prefs.setCharPref("extensions.shield-recipe-client.api_url", "${url.href}");\n` +
+          `Cu.unload('resource://shield-recipe-client/lib/RecipeRunner.jsm');\n` +
```

Comment:
I'm afraid I don't understand what problem this is trying to solve. Can you clarify what steps currently show an issue here?

At any point, there's supposed to be only one copy of any given jsm loaded in any one process, keyed by the URL you're using for the load. My understanding is that all of this code only runs in the parent process, and so does the code that updates add-ons. It's not clear to me how calling `Cu.unload` here makes a difference compared to `Cu.unload` being called with the exact same URL from the add-on's uninstall code. Are we sure that that uninstall code is running?